plugin-statusnotifier/statusnotifierwidget.cpp

```diff
@@ -28,6 +28,7 @@
 
 #include "statusnotifierwidget.h"
 #include <QApplication>
+#include <QDebug>
```

Comment:
There are debugs in this file -> https://github.com/lxde/lxqt-panel/blob/public_settings/plugin-statusnotifier/statusnotifierwidget.cpp#L40, https://github.com/lxde/lxqt-panel/blob/public_settings/plugin-statusnotifier/statusnotifierwidget.cpp#L53

I didn't introduce the qDebug() messages in this PR.... just the cleaning raised need of this include